packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -340,7 +340,10 @@ export interface IContextualMenuItem {
   * Method to provide the classnames to style the Vertical Divider of a split button inside a menu. Default value is the getVerticalDividerClassnames func defined in ContextualMenu.classnames
   * @default getSplitButtonVerticalDividerClassNames
   */
-  getSplitButtonVerticalDividerClassNames?: (theme: ITheme) => IVerticalDividerClassNames;
+  getSplitButtonVerticalDividerClassNames?: (theme: ITheme,
+    disabled: boolean,
```

Comment:
@dzearing I'm wondering if this is something that we should think about for the new styling API. When styling subcomponents, is it a common need to know the styling props of the component rendering the subcomponent?